src/app/operators/dialogs/csv/csv-config/csv-properties/csv-properties-template.component.html

```diff
@@ -286,7 +287,7 @@
         <form [formGroup]="layerProperties" style="width: 100%" fxLayout="column" fxLayoutAlign="space-around stretch">
             <mat-form-field style="width: 100%;">
                 <input matInput formControlName="layerName" placeholder="Layer Name"
-                       type="input" #layerNameComponent>
+                       type="input" #layerNameComponent id="layerNameComponent">
```

Comment:
Do you know if angular guarantees this id to be really a global id, i.e. resolves name conflicts in multiple components? Consider the case when another component also sets this id. For my understanding, an id must be globally unique.